lib/iris/plot.py

```diff
@@ -529,7 +533,8 @@ def _ensure_cartopy_axes_and_determine_kwargs(x_coord, y_coord, kwargs):
         cartopy_proj = ccrs.PlateCarree()
 
     # Ensure the current axes are a cartopy.mpl.GeoAxes instance.
-    _replace_axes_with_cartopy_axes(cartopy_proj)
+    if not kwargs.get('axes'):
+        _replace_axes_with_cartopy_axes(cartopy_proj)
```

Comment:
I'm not sure we can make this conditional - what if you've made an axes which isn't a cartopy GeoAxes and passed that through... it may be that we need to raise an exception in that situation.